layers/+lang/latex/README.org

```diff
@@ -97,6 +98,20 @@ To explicitly choose =company-auctex= as the backend, set the following in your
                 '((latex :variables latex-backend 'company-auctex)))
 #+END_SRC
 
+*** Company-math 
```

Comment:
This is just an addition to the company-auctex completion backend and must be part of this section please.

Reply:
Removed.